plugins/sysinfo/sysinfo.cpp

```diff
@@ -338,7 +340,10 @@ getWmiInfo (int mode)
 				hr = pclsObj->Get (L"Name", 0, &vtProp, 0, 0);
 				break;
 		}
-		WideCharToMultiByte (CP_ACP, 0, vtProp.bstrVal, -1, buffer, SysStringLen (vtProp.bstrVal)+1, NULL, NULL);
+		buffer_iter += WideCharToMultiByte(CP_ACP, 0, vtProp.bstrVal, -1, buffer_iter, SysStringLen(vtProp.bstrVal) + 1, NULL, NULL) - 1;
+		*(buffer_iter++) = ',';
```

Comment:
Here's what it outputs for me:

* King_Hual ** SysInfo ** Client: HexChat 2.10.2 (x64) ** OS: Microsoft Windows 7 Home Premium  ** CPU: Intel(R) Core(TM) i3 CPU       M 350  @ 2.27GHz (2.00 GHz) ** RAM: 3893 MB Total (1316 MB Free) ** VGA: AMD Radeon HD 6500M/5600/5700 Series, Intel(R) HD Graphics ** Uptime: 11.27 Hours **